src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
Sure, it is not many extra characters that are required but again 

> What bothers me the most is that the decisions of what sugar should be included and what should not feels very arbitrary.

to me `R.always(true)` is clearer than `R.T` and [propIs](https://github.com/ramda/ramda/blob/master/src/propIs.js#L27) also feels quite unneccesary.